packages/utils/typescript/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "chalk": "4.1.2",
     "cli-table3": "0.6.2",
-    "fs-extra": "10.0.1",
+    "fs-extra": "10.0.0",
```

Comment:
why the downgrade?

Reply:
Good catch. This also means Dependebot can upgrade them together again.